CONTRIBUTING.md

```diff
@@ -75,29 +80,78 @@ Note that the FLS ID should be filled according to the FLS paragraph ID for whic
 
 You would then pull `fls_4rhjpdu4zfqj` to place in the FLS ID field.
 
-### 2) Create a Draft with a Member
 
-A member of the Coding Guidelines Subcommittee should get you a first review with some feedback within 14 days of submission. You'll work with one or more members to flesh out the concept and ensure the guideline is well prepared.
+### 2) Guideline Generated as a Comment
+
+A GitHub Action will fire, adding a comment to your newly created issue with
+the contents of the coding guideline prepared written out correctly
+in reStructured Text.
+
+### 3) Create a Draft with a Member
+
+A member of the Coding Guidelines Subcommittee should get you a first review
+with some feedback within 14 days of submission. You'll work with one or more
+members to flesh out the concept and ensure the guideline is well prepared.
+
+### 4) Contributor Creates the PR
+
+Once an issue has been well-developed enough, a subcommittee member will mark
+the issue with the label `sign-off: create pr` to denote the contributor
+should create a PR based on the contents of the latest comment containing
+the generated guideline from
+[2) Guideline Generated as a Comment](#2-guideline-generated-as-a-comment).
```

Comment:
```suggestion
the issue with the label `sign-off: create pr` to denote that you
should create a Pull Request.

The contents of the PR should be based on the latest bot comment containing
the generated guideline from
[2) Guideline Generated as a Comment](#2-guideline-generated-as-a-comment).
```

The voicing is more personal elsewhere in the doc, so I'm mainly adjusting for that. The original sentence was also a bit long (it was a composite sentence), so I split it to make it easier to read.